python/lsst/ip/isr/ptcDataset.py

```diff
@@ -1013,9 +1025,9 @@ def appendPartialPtc(self, partialPtc):
                     self.covMatrixSide,
                 )
             )
-            self.covariancesModelNoB[ampName] = np.append(
-                self.covariancesModelNoB[ampName].ravel(),
-                partialPtc.covariancesModelNoB[ampName].ravel()
+            self._covariancesModelNoB[ampName] = np.append(
+                self._covariancesModelNoB[ampName].ravel(),
+                partialPtc._covariancesModelNoB[ampName].ravel()
```

Comment:
We aren't using this at all; I think if we need filler this should be explicitly filled with nans.